### DISCUSSION TITLE
Add readOnly prop support to Select and MultiSelect components

### DISCUSSION THREAD

**Discussion by mees-van-wel** [mantinedev/mantine#1399] [Feature requests]
Hi everyone,

Is it maybe an idea to add read only support to the select components.
I'm already using it in the `TextInput` wich works great with some custom styling. And I know with some workarounds I could just make use of the disabled prop and overwrite the disabled styling of the select components but maybe it would be nicer to automatically disable the logic without applying the disabled style when the `readOnly` prop is passed.

Btw you you've build a beautiful component library, much respect.

Mees

**Comment by rtivital** [comment]
Thanks, we will consider adding this feature
